JS-Algorithms/challenges.js

```diff
@@ -42,6 +51,13 @@ const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
   // YOUR CODE HERE...
+  if( isNaN(index) || index < 0 ) return null;
+
+  const newIndex = index % COUNTRY_NAMES.length;
+  const newArray = [ ...COUNTRY_NAMES, ...COUNTRY_NAMES ];
```

Comment:
can you think in a way of doing this without concatenating the same array twice?